### ISSUE TITLE
Recent changes don't work any more

### ISSUE THREAD

**Issue by marco-brandizi** [ljnelson/jpa-maven-plugin#8]
Recent changes in 3-SNAPSHOT (I understand they were done due to issue #6 and incompatibility with Java 8) don't work for me any more. I had to downgrade to version 2. The project at issue is this: https://github.com/EBIBioSamples/biosd_model. The plug-in generates a .properties file, but this contains header comments only, no property to define the list of classes.

I'm sorry I cannot provide further help, no idea what's going on.

**Comment by jlerbsc** [comment]
[WARNING] After scanning for Entities, no annotated Entities were found. on java 8 jpa project (jpa entitiy jar files are in the classpath).
com.edugility:jpa-maven-plugin:jar:3-20140611.160749-5
and dependencies 
[DEBUG]    net.sf.scannotation:scannotation:jar:1.0.2:compile
[DEBUG]    org.javassist:javassist:jar:3.18.2-GA:compile
Is this project dead?

**Comment by ljnelson** [comment]
No, not dead, but it sounds like you found an issue, likely with
Scannotation, which is ancient. Could you kindly file a bug?

**Comment by jlerbsc** [comment]
I'm sorry for the late response. I open an issue which is more a question 
https://github.com/ljnelson/jpa-maven-plugin/issues/11

**Comment by iSnow** [comment]
There is a "new" Scannotation version which is at least good enough for Java 8. Together with the latest Javassist version, the plugin works under Java 8 again.

Since it seems to be rather dead, I've forked it here: https://github.com/iSnow/jpa-maven-plugin
